css-nesting-1/Overview.bs

```diff
@@ -225,6 +225,72 @@ Direct Nesting {#direct}
 		   :is(.foo, .bar) + .baz,
 		   :is(.foo, .bar).qux { color: red; }
 		 */
+
+		.foo {
+			color: blue;
+			& .bar & .baz & .qux { color: red; }
+		}
+		/* equivalent to
+		   .foo { color: blue; }
+		   .foo .foo .bar .foo .baz .foo .qux { color: blue; }
+		 */
+
+		.foo {
+			color: blue;
+			& { padding: 2ch; }
+		}
+		/* equivalent to
+		   .foo { 
+		   	 color: blue; 
+		   	 padding: 2ch;
+		   }
```

Comment:
Would probably expand this more literally *first* (as two rules that both have `.foo` as the selector), then say it's also equivalent to this "both in one rule" example.